cmd/psqldef/tests.yml

```diff
@@ -162,6 +162,32 @@ AlterTypeAddValue:
     ALTER TYPE public.eventtype ADD VALUE 'send_meeting_options';
     ALTER TYPE public.eventtype ADD VALUE 'update_reactionslot';
   min_version: '12'
+AlterTypeAddValueWithSameTypeNameInDifferentSchema:
+  init: |
+    CREATE SCHEMA schema1;
+    CREATE SCHEMA schema2;
```

Comment:
The `init` is no longer needed as the `CREATE SCHEMA` is supported by the changes in https://github.com/sqldef/sqldef/pull/541. Put it directly in `current`.

Reply:
@hokaccha 
I fixed this at [a7975f3](https://github.com/sqldef/sqldef/pull/532/commits/a7975f3e997424837fb1445cadb2d69431e36f91) and [8cca999](https://github.com/sqldef/sqldef/pull/532/commits/8cca99938a0d680e48ac47a33acb64cf641a236b).